homeassistant/components/qwikswitch.py

```diff
@@ -0,0 +1,133 @@
+"""
+Support for Qwikswitch lights and switches.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/qwikswitch
+"""
+
+import logging
+from homeassistant.const import EVENT_HOMEASSISTANT_STOP
+from homeassistant.components.light import ATTR_BRIGHTNESS
+from homeassistant.components.discovery import load_platform
+
+REQUIREMENTS = ['https://github.com/kellerza/pyqwikswitch/archive/v0.1.zip'
+                '#pyqwikswitch==0.1']
+DEPENDENCIES = []
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'qwikswitch'
+
+
+class QSToggleEntity(object):
+    # pylint: disable=line-too-long
+    """Representation of a Qwikswitch Entiry.
+
+    Implement base QS methods. Modeled around HA ToggleEntity[1] & should only
+    be used in a class that extends both QSToggleEntity *and* ToggleEntity.
+
+    Implemented:
+     - QSLight extends QSToggleEntity and Light[2] (ToggleEntity[1])
+     - QSSwitch extends QSToggleEntity and SwitchDevice[3] (ToggleEntity[1])
+
+    [1] /home-assistant/home-assistant/blob/dev/homeassistant/helpers/entity.py
+    [2] /home-assistant/home-assistant/blob/dev/homeassistant/components/light/__init__.py  # NOQA
```

Comment:
Please make them even shorter, there is no need to have `/home-assistant/home-assistant/blob/dev/` here